Mathlib/NumberTheory/PythagoreanTriples.lean

```diff
@@ -508,39 +481,35 @@ theorem isPrimitiveClassified_of_coprime_of_odd_of_pos (hc : Int.gcd x y = 1) (h
     exact lt_add_of_pos_of_le zero_lt_one (sq_nonneg q)
   have hp : (⟨v, w⟩ : ℚ × ℚ) ∈ { p : ℚ × ℚ | p.1 ^ 2 + p.2 ^ 2 = 1 ∧ p.2 ≠ -1 } := ⟨hq, hw1⟩
   let q := (circleEquivGen hQ).symm ⟨⟨v, w⟩, hp⟩
-  have ht4 : v = 2 * q / (1 + q ^ 2) ∧ w = (1 - q ^ 2) / (1 + q ^ 2) :=
-    by
+  have ht4 : v = 2 * q / (1 + q ^ 2) ∧ w = (1 - q ^ 2) / (1 + q ^ 2) := by
     apply Prod.mk.inj
     have := ((circleEquivGen hQ).apply_symm_apply ⟨⟨v, w⟩, hp⟩).symm
     exact congr_arg Subtype.val this
-  let m := (q.denom : ℤ)
+  let m := (q.den : ℤ)
   let n := q.num
   have hm0 : m ≠ 0 := by
     norm_cast
     apply Rat.den_nz q
   have hq2 : q = n / m := (Rat.num_div_den q).symm
-  have hm2n2 : 0 < m ^ 2 + n ^ 2 :=
-    by
+  have hm2n2 : 0 < m ^ 2 + n ^ 2 := by
     apply lt_add_of_pos_of_le _ (sq_nonneg n)
     exact lt_of_le_of_ne (sq_nonneg m) (Ne.symm (pow_ne_zero 2 hm0))
-  have hw2 : w = (m ^ 2 - n ^ 2) / (m ^ 2 + n ^ 2) :=
-    by
-    rw [ht4.2, hq2]
-    field_simp [hm2n2, Rat.den_nz q, -Rat.num_div_den]
-  have hm2n20 : (m : ℚ) ^ 2 + (n : ℚ) ^ 2 ≠ 0 :=
-    by
+  have hm2n20 : (m : ℚ) ^ 2 + (n : ℚ) ^ 2 ≠ 0 := by
     norm_cast
     simpa only [Int.coe_nat_pow] using ne_of_gt hm2n2
-  have hv2 : v = 2 * m * n / (m ^ 2 + n ^ 2) :=
-    by
-    apply Eq.symm
-    apply (div_eq_iff hm2n20).mpr
-    rw [ht4.1]
-    field_simp [hQ q]
-    rw [hq2]
```

Comment:
This line is the one that mathport is complaining about, it was originally `rw [hq2] {occs := occurrences.pos [2, 3]},`